php/frameworks/Dockerfile

```diff
@@ -190,3 +190,29 @@ RUN set -xe; \
     git apply *.patch
 
 CMD [ "./run.sh" ]
+
+########################################################################################################################
+## phpredis/phpredis frameworks tests
+########################################################################################################################
+FROM base AS phpredis
+ARG PHPREDIS_VERSION_TAG
+
+RUN set -xe; \
+    git clone --depth 1 --branch ${PHPREDIS_VERSION_TAG} https://github.com/phpredis/phpredis.git \
+        /home/phpredis-${PHPREDIS_VERSION_TAG};
+
+WORKDIR /home/phpredis-${PHPREDIS_VERSION_TAG}
+
+ENV DD_SERVICE_NAME=phpredis-${PHPREDIS_VERSION_TAG}
```

Comment:
Nit: `DD_SERVICE` ;)

Reply:
Nice catch. Updated all the references in the same files. Ready for a second round of CR.